internal/codespaces/ssh.go

```diff
@@ -38,18 +38,35 @@ func Shell(ctx context.Context, p printer, sshArgs []string, port int, destinati
 func Copy(ctx context.Context, scpArgs []string, port int, destination string) error {
 	// Beware: invalid syntax causes scp to exit 1 with
 	// no error message, so don't let that happen.
-	cmd := exec.CommandContext(ctx, "scp",
+	connArgs := []string{
 		"-P", strconv.Itoa(port),
 		"-o", "NoHostAuthenticationForLocalhost=yes",
 		"-C", // compression
-	)
-	for _, arg := range scpArgs {
-		// Replace "remote:" prefix with (e.g.) "root@localhost:".
-		if rest := strings.TrimPrefix(arg, "remote:"); rest != arg {
-			arg = destination + ":" + rest
+	}
+
+	cmdArgs, command, err := parseSCPArgs(scpArgs)
```

Comment:
Given this is non-trivial, can we add a comment as we have in the `newSSHCommand` function? I realize it'll be very similar to what we say for SSH but a future reader might not know the similarities.

Reply:
Makes sense, added